app/forms/level_of_help_form.rb

```diff
@@ -9,4 +9,8 @@ class LevelOfHelpForm
 
   attribute :level_of_help
   validates :level_of_help, presence: true, inclusion: { in: LEVELS_OF_HELP }
+
+  def hide_asylum_and_immigration?(text)
+    text.downcase.include?("immigration") && !FeatureFlags.enabled?(:asylum_and_immigration)
```

Comment:
As it stands this code will break if we add Welsh translations (on the roadmap) before we enable the A&I feature flag. In general, having to know anything about the content of a user-facing text string is probably to be avoided.

In this instance I'd lean towards modifying en.yml so that the iffy bullet has its own key. So then we can do something more like:

```slim
  ul.govuk-list.govuk-list--bullet
    - t(".controlled_chambers").each do |bullet|
      li = bullet
    - if form.show_asylum_and_immigration_content?
      li =  t(".asylum_and_immigration_chamber")
```

Reply:
ah yes good point on the Welsh! Will amend 👍